redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -348,6 +348,16 @@ object Input {
     }
   }
 
+  case object ListMaxLenInput extends Input[ListMaxLen] {
+    override private[redis] def encode(data: ListMaxLen): Chunk[RespValue.BulkString] =
```

Comment:
I'd remove override private prefix, just to align it with othe inputs.

Reply:
done